src/Symfony/Component/Scheduler/Event/SchedulerEvents.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Scheduler\Event;
+
+final class SchedulerEvents
```

Comment:
I'm pretty sure we don't want this class alias system anymore. We just the use event's FQCN.